src/patternfly/components/AboutModalBox/about-modal-box.scss

```diff
@@ -118,33 +121,31 @@
   grid-template-areas:
     "brand close"
     "header header"
-    "content content"
-    "logo logo";
+    "content content";
 
   // Modal is full screen until larger breakpoint
   width: 100vw;
   height: 100vh;
   overflow-x: hidden;
   overflow-y: auto;
+  /* stylelint-disable */
+  scrollbar-color: #c10;
```

Comment:
🔥🔥🔥

should this be a var tho?

Reply:
bah, I don't want that just playing around with it. Forgot to take it out. It is cool though, if It worked haha.